admin-dev/themes/default/template/controllers/search/helpers/view/view.tpl

```diff
@@ -151,16 +151,24 @@ $(function() {
 
 {/if}
 <div class="row">
-    <div class="col-lg-6">
+    <div class="col-lg-{if $alternativeSearchPanels|@count == 0}6{else}4{/if}">
         <div class="panel">
             <h3>{l s='Search doc.prestashop.com' d='Admin.Navigation.Search'}</h3>
             <a href="https://doc.prestashop.com/dosearchsite.action?spaceSearch=true&amp;queryString={$query}&amp;utm_source=back-office&amp;utm_medium=search&amp;utm_campaign=back-office-{$lang_iso|upper}&amp;utm_content=download" class="btn btn-default _blank">{l s='Go to the documentation' d='Admin.Navigation.Search'}</a>
```

Comment:
@Julievrz / @atomiix Is it possible to use Gitbook rather than doc.prestashop.com ?